File: Beginner/go/1020.go
package main
 
import (
    "fmt"
)

func main() {
	var days, months, years int32;

	fmt.Scanln(&days);
	
	years = days / 365;
	months = (days % 365) / 30;
	days = (days % 365) % 30;
	
	fmt.Printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", years, months, days);
}

File: Beginner/cpp/1020.cpp
#include <iostream>
using namespace std;
 
int main() {
    int n, m, d, y;
    
    cin >> n;
    
    y = n / 365;
    m = n % 365 / 30;
    d = n % 365 % 30;
    
    // cout << y << " ano(s)\n";
    // cout << m << " mes(es)\n";
    // cout << d << " dia(s)\n";
    
    cout << y << " ano(s)\n" << m << " mes(es)\n" << d << " dia(s)\n";
 
    return 0;
}
